src/libs/actions/Report.js

```diff
@@ -425,6 +417,65 @@ function removeOptimisticActions(reportID) {
     });
 }
 
+/**
+ * If an iouReport is open (has an IOU, but is not yet paid) then we maintain a link between it and the associated
+ * chatReport in Onyx, simplifying IOU data retrieval and reducing necessary API calls when displaying IOU components:
+ * - chatReport: {id: 123, iouReportID: 987, ...}
+ * - iouReport: {id: 987, chatReportID: 123, ...}
+ *
+ * This function allows us to fetch an iouReport without updating the linked chatReport, preventing the chatReport's
+ * 'iouReportID' value from being updated. As an example, this is desired when fetching historical reports, to avoid
+ * overwritting an open iouReportID with a closed iouReportID. If this was to occur, unpaid IOUs would not be
+ * highlighted to the user.
+ *
+ * If updating the chatReport's data is desired, use `fetchIOUReportByIDAndUpdateChatReportLink` instead.
```

Comment:
NAB, the docs don't really feel like the right place for these comments and I'd move them to where we call `fetchIOUReportByIDAndUpdateChatReportLink()` or the docs for that method. 

We can see that this function fetches the iou report and updates the data. We do not need a lengthy justification for why this function only sets the IOU data.

Reply:
I just worry that someone will find this function first, parse the docs and not realize that they should actually be calling the other function. Perhaps this is an unnecessary fear though. I'll try reducing the comment and see how it looks.

Reply:
Removed the large comment.